### ISSUE TITLE
Minor accessibility a11y violation on InputWithMultiSelect

### ISSUE THREAD

**Issue by oliv-yu** [liferay/clay#2271]
It looks like Storybook picked up two a11y violations on this page with InputWithMultiSelect, didn't know if anyone had mentioned them already:

https://storybook.clayui.com/?path=/story/clayform--inputwithmultiselect

Thanks!

**Comment by matuzalemsteles** [comment]
Thanks @oliv-yu!

**Comment by kresimir-coko** [comment]
I've investigated this issue, checked all components and their variations in storybook and here's what I came up with - [Accessibility violations in Clay Storybook](https://docs.google.com/spreadsheets/d/1rrzP_H9XBF9pW6omBLpI22NYLjB7JO-6ZvM2P260Z44/edit#gid=0)

Requesting feedback as to how to proceed.

**Comment by bryceosterhaus** [comment]
@kresimir-coko looks like the doc is private. I'm unable to access it

**Comment by kresimir-coko** [comment]
@bryceosterhaus Should be fine now, I've edited the sharing permissions.

**Comment by bryceosterhaus** [comment]
@kresimir-coko thanks! After looking at the doc, I think a good first step would be to address any accessibility issues we are able to just through the stories. I think a handful of these could be fixed on the story side first and that any that rely on changes to the source of the components, we can add those next. That sound good to you?

**Comment by kresimir-coko** [comment]
@bryceosterhaus will do!
